sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,70 @@
 Version history
 ===============
 
+6.1.0 (2019 May)
+----------------
+
+New file formats:
+
+* BDV
+   - added a new reader for Big Data Viewer files
```

Comment:
This will need a format documentation page.

Reply:
PR: https://github.com/ome/bio-formats-documentation/pull/100